src/main/common/filter.c

```diff
@@ -103,13 +151,27 @@ void biquadFilterInit(biquadFilter_t *filter, float filterFreq, uint32_t refresh
 
     switch (filterType) {
     case FILTER_LPF:
+        // 2nd order Butterworth (with Q=1/sqrt(2)) / Butterworth biquad section with Q
+        // described in http://www.ti.com/lit/an/slaa447/slaa447.pdf
         b0 = (1 - cs) * 0.5f;
         b1 = 1 - cs;
         b2 = (1 - cs) * 0.5f;
         a0 = 1 + alpha;
         a1 = -2 * cs;
         a2 = 1 - alpha;
         break;
+    case FILTER_LPF1: {
```

Comment:
I think standard in Betaflight is to put opening bracket for block in switch onto a new line.